src/bad-code.py

```diff
@@ -578,3 +578,60 @@ def Funnel(p,d1,d2):
     
     #sys.stdout=sys.__stdout__
 """
+def create(path,archiveList,xFilesFactor=None,aggregationMethod=None,sparse=False,useFallocate=False):
+  # Set default params
+  if xFilesFactor is None:
+    xFilesFactor = 0.5
+  if aggregationMethod is None:
+    aggregationMethod = 'average'
+
+  #Validate archive configurations...

```

Comment:
E114 indentation is not a multiple of four (comment)
E265 block comment should start with '# '